gradle.properties

```diff
@@ -41,6 +43,7 @@ servlet_api_version = 2.5
 slf4j_version = 1.7.6
 sling_api_version = 2.9.0
 sling_base_version = 2.2.2
+sling_commons_json_version = 2.0.6
```

Comment:
It would be better to continue using Groovy's built-in JSONBuilder for consistency as used elsewhere in the codebase rather than adding this new dependency!

Reply:
Sure, I'll switch it